frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+import * as React from 'react';
+import { Row } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { useLeaflet } from 'react-leaflet';
+import { MAX_ZOOM } from 'constants/strings';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+const LinkMenu = styled(Row)`
+  background-color: #f2f2f2;
+  height: 35px;
+  width: 341px;
+  margin-left: -10px;
+  font-size: 14px;
+  font-family: ProximaNova, sans-serif;
+  padding: 10px;
+  a {
+    padding: 0 10px;
+    color: #1a5a96;
```

Comment:
This reminds me that I still haven't figured out if there is a way to access css variables in styled components, will try to remember to research this.

Reply:
Styled components has away the wrap the app with the theme provider that allows us to access the theme colors